packages/turf-square-grid/package.json

```diff
@@ -43,9 +43,6 @@
     "@types/tape": "*"
   },
   "dependencies": {
-    "@turf/boolean-intersects": "6.x",
-    "@turf/distance": "6.x",
-    "@turf/helpers": "6.x",
```

Comment:
isn't `@turf/helpers` still required here for types?

Reply:
Yeah from what I can tell `@turf/helpers` now gets brought in pretty much everywhere to help with the typescript side of things.

Reply:
Same story in this package.json as above - we need to make it align to the TS world view.